configure.ac

```diff
@@ -109,6 +109,12 @@ AS_IF([${LN_S} --relative symlinktest 2>/dev/null],
 #
 AC_DEFINE_UNQUOTED([UCX_CONFIGURE_FLAGS], ["$config_flags"], [UCX configure flags])
 
+#
+# Define path of ucx.conf configuration file
+#
+SYSCONFDIR="`eval echo $sysconfdir`"
```

Comment:
why need "eval"?
I think it doesn't work as needed when ./configure command has sys conf dir parameter

Reply:
`eval` is needed for evaluating sysconf=`${prefix}/etc`.

Reply:
Verified it works correctly for all combinations of: {with/without} x {prefix/sysconfdir} parasm.